src/main/features/core/windowTitleInfo.js

```diff
@@ -8,8 +8,14 @@ const updateDarwinTitle = (newTitle) => {
   }, 200);
 };
 
+// fix display of ampersands on windows
+const replaceAmpersands = string => (
+  process.platform === 'win32' ? string.replace('&', '&&&') : string
```

Comment:
I first did it with '&&' but it rendered as '_'.

I looked into it and if I'm understanding it correctly it's because windows strips ampersands (and others) from tooltips specifically - so it ends up needing more escapes.

Last paragraph of 'Displaying Text': [msdn](https://msdn.microsoft.com/en-us/library/windows/desktop/bb760250(v=vs.85).aspx#Tool_Text)